stdlib/3/subprocess.pyi

```diff
@@ -214,15 +222,21 @@ if sys.version_info >= (3, 3):
 
 class CalledProcessError(Exception):
     returncode = 0
-    cmd = ...  # type: str
-    output = b''  # May be None
+    # morally: _CMD
+    cmd = ...  # type: Any
+    # morally: Optional[_TXT]
+    output = ...  # type: Any
 
     if sys.version_info >= (3, 5):
-        stdout = b''
-        stderr = b''
+        # morally: Optional[_TXT]
+        stdout = ...  # type: Any
+        stderr = ...  # type: Any
 
-    def __init__(self, returncode: int, cmd: str, output: Optional[str] = ...,
-                 stderr: Optional[str] = ...) -> None: ...
+    def __init__(self,
+                 returncode: int,
+                 cmd: _CMD,
+                 output: Optional[_TXT] = ...,
```

Comment:
Odd that this argument isn't called "stdout", but confirmed in the source that it's true.